test/unit/FutarchyChallenge.js

```diff
@@ -0,0 +1,259 @@
+import lkTestHelpers from 'lk-test-helpers'
+const { expectRevert } = lkTestHelpers(web3)
+
+const FutarchyOracleFactoryMock = artifacts.require('FutarchyOracleFactoryMock.sol')
+const FutarchyOracleMock = artifacts.require('FutarchyOracleMock.sol')
+const FutarchyChallenge = artifacts.require('FutarchyChallenge.sol')
+const Token = artifacts.require('EIP20.sol')
+const BigNumber = require('bignumber.js');
+
+contract('FutarchyChallenge', (accounts) => {
+
+  const mockAddresses = [
+    '0x4e0100882b427b3be1191c5a7c7e79171b8a24dd',
+    '0x6512df5964f1578a8164ce93a3238f2b11485d1c',
+    '0x687355ca7a320e5420a3db5ae59ef662e4146786'
+  ]
+
+  const REGISTRY_ADDR         = mockAddresses[0]
+  const CHALLENGER            = accounts[0]
+  const LISTING_OWNER         = accounts[1]
+  const STAKE_AMOUNT          = 10 * 10 ** 18
+  const TRADING_PERIOD        = 60 * 60 * 24
+  const LOWER_BOUND           = 0
+  const UPPER_BOUND           = 100
+  const LMSR_ADDR             = mockAddresses[1]
+  const SCALAR_P_ORACLE_ADDR  = mockAddresses [2]
```

Comment:
does that say poracle!?

Reply:
@jpkcambridge lol

@emaG3m might be better to make it `SCALAR_PRICE_ORACLE_ADDR`, even though it's long